QuizAppNew/View/QAview.swift

```diff
@@ -10,6 +10,12 @@ import SwiftUI
 //MARK: Question Answer view
 struct QAview: View {
     @State var isSubmitted = false
+    @ObservedObject var viewModel = QuestionViewModel()
+    //outgoing
+    @State var correct: Int = 0
+    @State var wrong: Int = 0
+    @State var answered: Int = 0
+    
```

Comment:
why  **correct ,  wrong, and answered** variable declared here?

Reply:
removed

Reply:
why removed? why did you declare?

Reply:
@State var read and write kore value manage kore .. ekahne correct , wrong, and answered er value ta state property wrapper er maddhome value manage korbe.. remove koresi cz ekahne view dekhnor dorkar nei ekhon shudu json load kortesi